_sass/pages/_blog.scss

```diff
@@ -0,0 +1,7 @@
+// Unique elements for blog pages
+
+.blog-page {
+  table.big-query {
+    @include tables(200px);
```

Comment:
Great point.  I have removed the _blog.scss file entirely and created a separate components/_tablse.scss file.  In doing so, this removed the ``blog-page`` from the selector, so I think this will be more logical?

Addressed comment in https://github.com/m-lab/m-lab.github.io/pull/91/commits/cfd1724768b3c1f78fd772dc279e3cde618b3d6b